chef_master/source/ruby.rst

```diff
@@ -612,10 +612,8 @@ Constructs to Avoid
 -----------------------------------------------------
 Avoid the following patterns:
 
-* ``node.set`` / ``normal_attributes`` - Avoid using attributes at normal precedence since they are set directly on the node object itself, rather than implied (computed) at runtime.
-* ``node.set_unless`` - Can lead to weird behavior if the node object had something set. Avoid unless altogether necessary (one example where it's necessary is in ``node['postgresql']['server']['password']``)
+* ``node.normal` - Avoid using attributes at normal precedence since they are set directly on the node object itself, rather than implied (computed) at runtime.
```

Comment:
```suggestion
* ``node.normal`` - Avoid using attributes at normal precedence since they are set directly on the node object itself, rather than implied (computed) at runtime.
```